frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -349,5 +355,13 @@ export const eventUsageLogic = kea<
         reportProjectHomeSeen: async ({ teamHasData }) => {
             posthog.capture('project home seen', { team_has_data: teamHasData })
         },
+        reportEventSearched: async ({ search_term, extra_props }) => {
+            if (preflightLogic.values.realm !== 'cloud') {
```

Comment:
do we still need to return something if we change the condition to === 'cloud' and move the capture on line 364 to here instead?

Reply:
good catch!